src/main/java/org/owasp/benchmark/score/parsers/HdivReader.java

```diff
@@ -65,21 +66,29 @@ else if (line.contains("Product version:")) {
 
 	private String calculateTime(final String firstLine, final String lastLine) {
 		try {
-			String start = firstLine.split(" ")[0];
-			String stop = lastLine.split(" ")[0];
-			SimpleDateFormat sdf = new SimpleDateFormat("HH:mm:ss,SSS");
-			Date startTime = sdf.parse(start);
-			Date stopTime = sdf.parse(stop);
-			long startMillis = startTime.getTime();
-			long stopMillis = stopTime.getTime();
-			return (stopMillis - startMillis) / 1000 + " seconds";
+			try {
+				return calculateTime(firstLine, lastLine, 0);
+			} catch (ParseException e) {
+				return calculateTime(firstLine, lastLine, 1);
+			}
 		}
 		catch (Exception e) {
 			e.printStackTrace();
 		}
 		return null;
 	}
 
+	private String calculateTime(final String firstLine, final String lastLine, final int timeColumn) throws ParseException {
+		String start = firstLine.split(" ")[timeColumn];
+		String stop = lastLine.split(" ")[timeColumn];
+		SimpleDateFormat sdf = new SimpleDateFormat("HH:mm:ss,SSS");
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The SimpleDateFormat object was created without setting the time zone. Make sure that you want to use the default time zone or use letter z, Z or X in the pattern. Otherwise, if you do not call setTimeZone() on the created SimpleDateFormat object, you might get an unexpected date and time when using the object.